### PR TITLE
WF-57277-Made UG changes to add missed statements

### PR DESCRIPTION
**Pull request by elizabethatienoo**
<table style="width: 521.306px;">
<tbody>
<tr>
<td style="width: 156px;">Task Link</td>
<td style="width: 349.306px;">(https://syncfusion.atlassian.net/browse/WF-57277)</td>
</tr>
<tr>
<td style="width: 156px;">Description</td>
<td style="width: 349.306px;">Added Blazor Server side in Word to PDF namespace list in DocIO UG</td>
</tr>
<tr>
<td style="width: 156px;">Solution</td>
<td style="width: 349.306px;">Added Blazor Server side in Word to PDF namespace list, Added license clearance information to use Microsoft fonts, Added a note in UG about how to insert multiple body items using mergefield event handler</td>
</tr>
<tr>
<td style="width: 156px;">Content review confirmation</td>
<td style="width: 349.306px;">Yes</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: File-Formats/DocIO/FAQ.md [hunk 1]
```diff
@@ -2368,6 +2368,8 @@ sudo apt-get install ttf-mscorefonts-installer
 
 After the installation, the necessary Microsoft compatible fonts will be available in this location "/usr/share/fonts/truetype/msttcorefonts", which will be considered for conversion.
 
+N> To use Microsoft fonts in your environment, kindly get license clearance from Microsoft at your side.
```
Comment (Chrispineagunja) [inline File-Formats/DocIO/FAQ.md:2371]: N> To use Microsoft fonts in your environment, kindly get the license clearance from Microsoft on your side.

File: File-Formats/DocIO/mail-merge/mail-merge-events.md [hunk 2]
```diff
@@ -145,6 +145,8 @@ private void ApplyAlternateRecordsTextColor (object sender, MergeFieldEventArgs
 
 {% endtabs %}
 
+N> While executing mail merge, DocIO internally uses copy of particular region for populating the contents. Sometimes, it may rise unexcepted problems due to inserting multiple body items in to the region through mail merge process. So, to insert multiple body items using mergefield event handler, we recommend you to use this [approach](https://www.syncfusion.com/kb/11701/how-to-replace-merge-field-with-html-string-using-mail-merge) at your side.
```
Comment (Chrispineagunja) [inline File-Formats/DocIO/mail-merge/mail-merge-events.md:148]: N> While executing mail merge, DocIO internally uses a copy of a particular region for populating the contents. Sometimes, unexpected problems may arise due to inserting multiple body items into the region through the mail merge process. So, to insert multiple body items using the mergefield event handler, you are recommended to use this [approach](https://www.syncfusion.com/kb/11701/how-to-replace-merge-field-with-html-string-using-mail-merge) at your side.

File: File-Formats/DocIO/word-to-pdf.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ The following namespaces are required to compile the code:
 * using Syncfusion.OfficeChartToImageConverter
 * using Syncfusion.Pdf
 
-**For ASP.NET Core and Xamarin applications**
```
Comment (Chrispineagunja) [inline File-Formats/DocIO/word-to-pdf.md:30]: **For ASP.NET Core, Blazor, Xamarin, WinUI, and .NET MAUI applications**

File: File-Formats/DocIO/mail-merge/mail-merge-events.md [hunk 4]
```diff
@@ -145,6 +145,8 @@ private void ApplyAlternateRecordsTextColor (object sender, MergeFieldEventArgs
 
 {% endtabs %}
 
+N> N> While executing mail merge, DocIO internally uses a copy of a particular region for populating the contents. Sometimes, unexpected problems may arise due to inserting multiple body items into the region through the mail merge process. So, to insert multiple body items using the mergefield event handler, you are recommended to use this [approach](https://www.syncfusion.com/kb/11701/how-to-replace-merge-field-with-html-string-using-mail-merge) at your side.
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/mail-merge/mail-merge-events.md:148]: You wrongly added two note. Please remove one N>

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**